app/src/androidTest/java/org/mozilla/focus/activity/ToggleBlockTest.java

```diff
@@ -73,10 +71,22 @@ public void tearDown() throws Exception {
     @Test
     public void SimpleToggleTest() throws UiObjectNotFoundException {
         // Load mozilla.org
-        onView(withId(R.id.url_edit))
-                .check(matches(isDisplayed()))
-                .check(matches(hasFocus()))
-                .perform(typeText("mozilla"), pressImeActionButton());
+        //
+        //onView(withId(R.id.url_edit))
+        //        .check(matches(isDisplayed()))
+        //        .check(matches(hasFocus()))
+        //        .perform(typeText("mozilla"), pressImeActionButton());
+        // Above types "mmozilla"
+        TestHelper.inlineAutocompleteEditText.waitForExists(waitingTime);
+        TestHelper.inlineAutocompleteEditText.clearTextField();
+        TestHelper.inlineAutocompleteEditText.setText("mozilla");
+        TestHelper.hint.waitForExists(waitingTime);
+        TestHelper.pressEnterKey();
+        TestHelper.webView.waitForExists(waitingTime);
```

Comment:
For in-app tests I would prefer if we switch to Espresso which is way more powerful instead of going back to uiautomator.